tests/curators/test_curators_multivalue.py

```diff
@@ -0,0 +1,48 @@
+import bionty as bt
+import lamindb as ln
+import pandas as pd
+import pytest
+from lamindb.core.exceptions import ValidationError
+
+
+@pytest.fixture
+def df():
+    return pd.DataFrame(
+        {
+            "sample_id": [["sample1", "sample2"], ["sample2"], ["sample3"]],
+            "dose": [[1.2, 2.3], [1.2], [2.3]],
+            "cell_type": [["B cell", "T cell"], ["B cell"], ["T cell"]],
+            "tissue": [["blood", "pulmo"], ["blood"], ["lung"]],
+        }
+    )
+
+
+@pytest.fixture(scope="module")
+def lists_schema():
+    schema = ln.Schema(
+        name="lists schema cat",
+        features=[
+            ln.Feature(name="sample_id", dtype=str).save(),
+            ln.Feature(name="dose", dtype=float).save(),
+            ln.Feature(name="cell_type", dtype=str).save(),
+            ln.Feature(name="tissue", dtype=bt.Tissue).save(),
```

Comment:
This should be typed as `list[bt.Tissue]`.

Reply:
No, it will be flexible.

Reply:
See comment below. It should be typed as `list[bt.Tissue]`.

Reply:
No, we can't make two features both named "tissue"; one of dtype `cat[bt.Tissue]` and the other is `list[cat[bt.Tissue]]`. This will drive users crazy if they can't use they have to make a different `tissue` feature.

Reply:
Yes, we cannot. But we have to be able to error in case the user wishes to avoid lists in a feature.